xdsl/dialects/llvm.py

```diff
@@ -1703,7 +1703,12 @@ def __init__(
         input_types = [
             SSAValue.get(arg).type for arg in args[: len(args) - variadic_args]
         ]
-        var_callee_type = LLVMFunctionType(input_types, return_type, variadic_args > 0)
+        if variadic_args > 0:
```

Comment:
We generally go for truthiness, except for when checking for `None`
```suggestion
        if variadic_args:
```